src/Prefabs/Vegetation/Tree/Dst/Tree_Dst.et

```diff
@@ -0,0 +1,63 @@
+GenericEntity {
+ ID "5A631817098CF41D"
+ components {
+  MeshObject "{5A6318165B9E4A17}" {
+   Object "{E51D1C3DB7D7DD93}Assets/Vegetation/Tree/Pinus_Sylvestris/Dst/t_pinus_sylvestris_2f_dst_base.xob"
```

Comment:
Could we maybe do some trickery to spawn this destruction with the same mesh object as the source? Maybe we can get and set them via script? Or was that not the plan because those all have tiny branches on them?

Even so, I would probably make this base class with a mesh component but no object set and then see if we add a few common variants for the very distinct tree colors. So a nearly white tree `{616D6371AEB1D799}Prefabs/Vegetation/Tree/t_betula_pendula_2t.et` does not spawn the same log when being cut down as a "brown" one?

Reply:
Yeah I wanted a "clean" log for the destruction.

I didn't want to override every single tree with the fitting destruction mesh.
Im waiting for the TreeV2 update with the vanilla destruction (and fitting destruction meshes).

Reply:
Ok whatever you want to do in the future make an issue about it then so we do not forget :)

Reply:
https://github.com/EveronLife/EveronLife/issues/172